python/mxnet/_numpy_op_doc.py

```diff
@@ -176,3 +176,116 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         `axis` is not None or `a` is a 1-d array.
     """
     pass
+
+def _np_prod(a, axis=None, dtype=None, out=None, keepdims=None):
```

Comment:
Please review the order of the parameters. The third argument should be ``keepdims``